src/components/teasers/teaser.tsx

```diff
@@ -89,12 +90,13 @@ export const Teaser: React.FC<TeaserProps> = ({
   dateFormatted,
   cover,
   linkTo,
+  className,
   children,
 }) => {
   const hasToplineContainer = Boolean(topline || dateFormatted);
 
   return (
-    <TeaserContainer>
+    <TeaserContainer className={className}>
```

Comment:
styled component support again. I wanted to modify a single Teaser (`TeaserHighlighted`) hence this change which is also good for the future